### PR TITLE
update docs

### PR DESCRIPTION
**Pull request by t-bltg**
Following https://github.com/korsbo/Latexify.jl/pull/225.

### CODE REVIEW

File: docs/src/index.md [hunk 1]
```diff
@@ -237,18 +237,26 @@ Alternatively, `render(str, mime)` can also be used to generate and display DVI,
 latexify(:(x/y)) |> s -> render(s, MIME("image/png"))
 ```
 
-PNG and SVG outputs rely on [dvipng](http://www.nongnu.org/dvipng/) and [dvisvgm](https://dvisvgm.de/), respectively.
+PNG output relies on [ghostscript](https://www.ghostscript.com) or alternatively on [dvipng](http://www.nongnu.org/dvipng) (use the `convert` argument).
```
Comment (gustaphe) [inline docs/src/index.md:240]: What determines the "alternatively" part? What does "use the `convert` argument" refer to?

Comment (t-bltg) [reply docs/src/index.md:240]: https://github.com/korsbo/Latexify.jl/blob/956eccb5982244649087c909f2895400b03d6cb5/src/utils.jl#L154

and 

https://github.com/korsbo/Latexify.jl/blob/956eccb5982244649087c909f2895400b03d6cb5/src/utils.jl#L190

Comment (t-bltg) [reply docs/src/index.md:240]: Maybe I should add: use the _`convert` keyword argument of the function `render` instead_ ?

Comment (gustaphe) [reply docs/src/index.md:240]: Yes, I'm thinking from the perspective of someone reading the documentation. Perhaps something like "if `render` is called with `convert=:gs` then ..."

**Review by korsbo — APPROVED** [review]

**Pull request merged**